content/200-concepts/100-components/01-prisma-schema/06-relations/425-referential-actions/100-cyclic-referential-actions.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: 'Cyclic referential actions'
+metaTitle: 'Cyclic referential actions'
+metaDescription: 'Circular references can occur between models when each model references another creating a closed loop. Learn how Prisma identifies and suggests solutions to this potential problem.'
```

Comment:
```suggestion
metaDescription: 'Circular references can occur between models when each model references another creating a closed loop. Since Microsoft SQL Server does not handle these situations out of the box, learn how to solve this problem.'
```